Classes/PostProcessor/SaveFormPostProcessor.php

```diff
@@ -161,9 +161,13 @@ public function process() {
     protected function processFields($formElements, &$text, $submittedValues) {
         foreach ($formElements as $element) {
             if(is_a($element,'TYPO3\\CMS\\Form\\Domain\\Model\\Element\\ContainerElement')) {
-                if($element->getAdditionalObjectByKey('legend')->getValue()) {
-                    $text .= $element->getAdditionalObjectByKey('legend')->getValue() . "\n";
-                }
+                
+                if ( !empty($element->getAdditionalObjectByKey('legend')) ) {
```

Comment:
remove empty spaces inside bracket of "if". and better to check it's object maybe ?